src/Microsoft.OData.Client/DataServiceContext.cs

```diff
@@ -692,6 +700,12 @@ internal ClientEdmModel Model
             get { return this.model; }
         }
 
+        /// <summary>
+        /// Gets IsIeee754Compatible header value.
+        /// </summary>
+        internal bool IsIeee754Compatible { get; private set; }

```

Comment:
I am wondering why do we need to save "Ieee...Compatible" in a ServiceContext?

A service context is a context for all potential request. The state should not change. Now, you have to change this member every time for every request. I don't think it's a good place for it. 
@Michael Pizzo

Reply:
@mikepizzo Do you have any reservations on this?